lib/bpk-component-badge/src/BpkBadge.js

```diff
@@ -186,7 +186,7 @@ const BpkBadge = (props: Props) => {
   viewStyle.push(viewStyleMap[type]);
   textStyle.push(textStyleMap[type]);
 
-  const themeAttributes = {
+  const themeAttributes: Object = {
```

Comment:
Set this as an Object here as Flow was complaining that the values `themeAttributes.xxx` - does not exist in themeAttributes, I tried to make this object exact (see below) but Flow threw errors on the `getThemeAttributes` line afterwards

```

themeAttributes: {
  badgeSuccessBackgroundColor: string, 
  badgeSuccessTextColor: string, 
  ...
}
```